src/client/helpers/utils.js

```diff
@@ -34,4 +34,24 @@ function injectDefaultAliasName(instance) {
 	return instance;
 }
 
+function formatDate(date, includeTime) {
+	'use strict';
+	if (!date || !(Object.prototype.toString.call(date) === '[object Date]') ||
```

Comment:
as brought up by @leftmost, this condition is just a more reliable way of checking the object prototype since date objects do not have the same prototype across different frames. Granted, this scenario is unlikely to pop up. I'll add in a comment explaining purpose of this condition.